### PR TITLE
fix a wrong url in code-repositories.md

### PR DESCRIPTION
**Pull request by gzliudan**

### CODE REVIEW

File: developers/code-repositories.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Yearn smart contracts repo: [https://github.com/yearn/yearn-protocol](https://gi
 - [Registries](https://github.com/yearn/yearn-protocol/tree/develop/contracts/registries)
 - [Vaults](https://github.com/yearn/yearn-protocol/tree/develop/contracts/vaults)
 - [Strategies](https://github.com/yearn/brownie-strategy-mix)
-- [iToken Wrappers](https://github.com/yearn/brownie-strategy-mix)
+- [iToken Wrappers](https://github.com/yearn/itoken)
```
Comment (lehnberg) [inline developers/code-repositories.md:26]: Actually, would be even better if we linked to:

```
[Token Wrappers](https://github.com/yearn/brownie-wrapper-mix)
```
here instead, as far as I know the iToken repo is not active any longer.

**Review by lehnberg — CHANGES_REQUESTED** [review]
Thanks for the PR @gzliudan! Please see suggested changes above.

File: developers/code-repositories.md (hunk 1 above)
Comment (gzliudan) [reply developers/code-repositories.md:26]: I saw Strategies is linked to same link, so I think the link of iToken Wrappers is wrong. Sorry to trouble you.

**Comment by lehnberg** [comment]
nothing to be sorry for @gzliudan, feel free to push a fixed commit and I'll merge it!

**Comment by gzliudan** [comment]
Ok, the link is changed to [Token Wrappers](https://github.com/yearn/brownie-wrapper-mix) from [iToken Wrappers](https://github.com/yearn/brownie-strategy-mix) according to your suggest. I'm new to yearn. Thanks for your help.

**Comment by gzliudan** [comment]
Ok, the link is changed to [Token Wrappers](https://github.com/yearn/brownie-wrapper-mix) from [iToken Wrappers](https://github.com/yearn/brownie-strategy-mix) according to your suggest. I'm new to yearn. Thanks for your help.

**Review by lehnberg — APPROVED** [review]
Looks great, thanks for this, and welcome to Yearn! 👍

**Pull request merged**